host_vars/OCT-CORE-1/interfaces.yaml

```
@@ -21,30 +21,33 @@ interfaces:
     portmode: "access"
     mtu: 9216
   1/1/4:
-    description: "OCP-CTRL-U33L"
+    description: "ESI-CTRL-U33L"
     state: "up"
-    untagged: 10
+    untagged: 127
     tagged:
-      - 210
+      - 250
+      - 3801
     portmode: "hybrid"
     mtu: 9216
   1/2:
     fanout: "quad"
     fanout_speed: "25G"
   1/2/1:
-    description: "OCP-CTRL-U32L"
+    description: "ESI-CTRL-U32L"
```

```suggestion
    description: "ESI-CTRL-U32-L"
```

For consistency with R